v2/tools/generator/internal/codegen/pipeline/recursivetypefixer/simple_recursive_type_fixer.go

```diff
@@ -102,7 +101,6 @@ func (s *SimpleRecursiveTypeFixer) unrollRecursiveReference(this *astmodel.TypeV
 		return astmodel.IdentityVisitOfTypeName(this, it, ctx)
 	}
 
-	klog.V(3).Infof("%q references itself, removing the self-reference by unrolling to %q", typedCtx.name, typedCtx.unrolledName)
```

Comment:
I think this might make sense to keep as a verbose log still. The whole "recursive types" thing is awkward and might need to be debugged, and if it did presumably this log line would be useful?